### ISSUE TITLE
Sudden drop in test accuracy

### ISSUE THREAD

**Issue by nitika-verma** [pyg-team/pytorch_geometric#791]
## ❓ Questions & Help

Hi Matthias, Great work on the library!

I have been trying to run faust.py in the examples folder and came across a problem where the test accuracy suddenly drops and the training loss explodes to a large value. Is this something which you have observed happening before and could you help resolve it?

Thanks! 

![Screenshot at 2019-11-07 15:10:23](https://user-images.githubusercontent.com/2107799/68395837-ccabd680-0170-11ea-85b4-577b5831098e.png)

**Comment by rusty1s** [comment]
Yes, I sadly noticed that myself. I tried various things to identify the error without any luck. I believe there may be a numerical instability in the `SplineConv` implementation.

**Comment by sofitiwari** [comment]
Hi, faced same issue with faust.py example. Running the example with reduced learning rate seems to be working for all epochs (i used initial lr= 0.001, from epoch 61 onwards used lr=0.0001).

**Comment by tangisxx** [comment]
Hi, I met the same issue that the test accuracy suddencly dropped after a few epochs and the critical timing varies from time to time. Is there any suggestion on how to better implement the SplineConv layer?

Many thanks!

**Comment by rusty1s** [comment]
`grad_norm` seems to help in this regard.

**Comment by DanielFPerez** [comment]
Hi, 
I'm experiencing the same behavior with GATConv. A sudden drop in accuracy, precision, and recall.
I'm using adam, weighted x-entropy loss, init learning rate of 1e-3 with 5% decay every 30 epochs.
